### DISCUSSION TITLE
[Github] 깃허브(github) 프로필 꾸미기: 잔디 먹는 뱀

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#28] [4. 기타 글 모음]
앞으로 뱀 먹이 주려고 잔디를 열심히 심어야 할 거 같네요..ㅎ

다음 포스팅은 서류가 합격한다면 다음 주제는 이력서 적으면서 고민했던 내용들 및 레퍼런스를 적고 아니면 프로젝트 및 과제하면서 생각나는 주제를 써보겠습니다.

[블로그 링크](https://donkeysdevelpment.tistory.com/113)

**Comment by publdaze** [comment]
우와! 저도 얼마전에 잔디 먹는 뱀 발견해서 적용해보고 싶었는데 여기서 보다니 반갑네요!! 따로 찾아볼 필요없이 이 글 보면 되겠어요! 👍👍

**Reply by pnudpwls1103** [reply]
도움이 됐길 바래용 ㅎㅎ

**Comment by shkisme** [comment]
뱀 너무 귀엽다ㅏ 🐍  다음 주제도 기다릴게요~

**Reply by pnudpwls1103** [reply]
ㅎㅎ 다음 게시글도 유익하고 재밌는 글 가져와볼게용~!~!

**Comment by pnudpwls1103** [comment]
3개월 전 발생한 에러를 고쳐서 글에 업데이트 했습니다^^
(귀찮았는데 드디어 했다)

**Comment by 1jeongg** [comment]
앗 저도 리드미에 뱀 한 마리 키우고 있는데 한 번 내용이 바뀌었나? 그래서 갑자기 안됐던 걸로 기억해요 ㅠㅠ 뱀 있으니까 확실히 동기부여는 되는 것 같은데 깃허브 액션을 쓰면 PR이었나? 뭐가 초기화돼서 github stat 레벨이 내려가서 지웠던게 기억에 남네요

**Reply by pnudpwls1103** [reply]
헉 stat 확인을 안 해봐서 저는 문제 있는지 잘 모르겠네요 흠.. 다음에 관찰해보고 문제 없으면 댓글로 말씀드릴게요!!
